backbone/src/main/java/org/researchstack/backbone/model/survey/SurveyItem.java

```diff
@@ -35,18 +35,6 @@
     @SerializedName("items")
     public List<T> items;
 
-    @SerializedName("skipIdentifier")
-    public String skipIdentifier;
-
-    @SerializedName("skipIfPassed")
-    public boolean skipIfPassed;
-
-    // TODO: implement this?
-    public String rulePredicate;  // this is an NSPredicate on iOS, how do we convert?
```

Comment:
Maybe we can use this https://developer.android.com/reference/java/util/function/Predicate.html

And have a util class that parses a string and returns a Predicate?